api/src/Page/Download.php

```diff
@@ -474,7 +476,11 @@ function _streamZipFile($files, $zipName)
             $zip = new ZipStream($zipName . '.zip', $options);
             foreach ($files as $file) {
                 $filename = $file['FILEPATH'] . '/' . $file['FILENAME'];
-                $zip->addFileFromPath(basename($filename), $filename);
+                if (file_exists($filename)) {
+                    $zip->addFileFromPath(basename($filename), $filename);
+                } elseif (file_exists($filename.'.gz')) {
+                    $zip->addFileFromPath(basename($filename), $filename.'.gz');
```

Comment:
```suggestion
                    $zip->addFileFromPath(basename($filename).'.gz', $filename.'.gz');
```

Reply:
Rats, I wondered why we didn't get a zip of zips. Have committed the suggestion.